examples/compiler_singlepass.rs

```diff
@@ -63,4 +96,4 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 #[cfg(feature = "singlepass")]
 fn test_compiler_singlepass() -> Result<(), Box<dyn std::error::Error>> {
     main()
-}
+}
```

Comment:
Let's remove the changes to this file, the original intention was to serve as an example, re-using for benchmaring increases the amount of tech debt.

Reply:
OK